python-frontend/src/main/java/org/sonar/python/types/TypeShed.java

```diff
@@ -470,7 +470,13 @@ public static boolean isValidForProjectPythonVersion(List<String> validForPython
         symbols.add(fromOverloadedFunction(((OverloadedFunctionSymbol) descriptor), isInsideClass));
       }
       if (descriptor instanceof SymbolsProtos.VarSymbol) {
-        symbols.add(new SymbolImpl((SymbolsProtos.VarSymbol) descriptor));
+        SymbolsProtos.VarSymbol varSymbol = (SymbolsProtos.VarSymbol) descriptor;
+        SymbolImpl symbol = new SymbolImpl(varSymbol);
+        if (varSymbol.getIsImportedModule()) {
+          Set<Symbol> moduleExportedSymbols = searchTypeShedForModule(varSymbol.getFullyQualifiedName());
+          moduleExportedSymbols.forEach(symbol::addChildSymbol);
```

Comment:
I don't think we should copy the symbols here as it would create a lot of them, but are we sure we don't risk anything by creating so many links between existing instances?
Probably not and it certainly won't matter once we move to descriptors, but just wanted to point it out to make sure.

Reply:
I think they are already copied when we actually import module symbols inside the local symbol table (inside Scope#copySymbol), so I'd say we don't need to copy them here. Does it make sense?